README.md

```diff
@@ -124,11 +126,27 @@ To test local modifications, use pip's editable mode.
 To run unit tests under `test` folder:
 
 ```bash
-coverage run -m unittest discover
+make test
```

Comment:
I have to use `pymake` in Windows. <https://pypi.org/project/py-make/>

Reply:
I didn't think about that. Should we find another tool? I think the standard option is to run commands by hand.

Reply:
Should we instead use a containerized development environment to simplify accounting for various operating system development tools?

Reply:
Nah, not worth it, I was reading around and this is pretty standard practice for python development, plus the workflow works fine on Windows. I'm happy if you just make a side note to use `pymake` for Win.